contracts/NonfungibleTokenPositionDescriptor.sol

```diff
@@ -11,7 +13,21 @@ import './libraries/NFTDescriptor.sol';
 
 /// @title Describes NFT token positions
 /// @notice Produces a string containing the data URI for a JSON metadata string
-contract NonfungibleTokenPositionDescriptor is INonfungibleTokenPositionDescriptor {
+contract NonfungibleTokenPositionDescriptor is INonfungibleTokenPositionDescriptor, Initializable {
+    struct TokenRatioOrderPriority {
+        address token;
+        int256 priority;
+    }
+
+    // tokens that take priority order in price ratio - higher integers get numerator prioriy
+    mapping(address => int256) public tokenRatioPriority;
+
+    function initialize(TokenRatioOrderPriority[] memory tokens) public initializer() {
+        for (uint256 i = 0; i < tokens.length; i++) {
+            tokenRatioPriority[tokens[i].token] = tokens[i].priority;
```

Comment:
maybe should trigger an event for each token to have a record of which tokens were recorded...(helpful for making changes in the future)